src/components/Devices/DeviceListing.vue

```diff
@@ -1,40 +1,40 @@
 <template>
   <div>
-
-    <spinner :fetching="!fetched"/>
+    <spinner :fetching="!fetched" />
 
     <p v-if="!devices.length && fetched">You currently have no devices.</p>
 
     <div
       v-for="(device, index) in orderedDevices"
       :key="index"
       :class="fetchable()"
-      class="device">
+      class="device"
+    >
       <router-link
-        :to="{ name: 'device', params: { devicename: device.devicename }}"
+        :to="{ name: 'device', params: { devicename: device.devicename } }"
         tag="div"
-        class="header">
-
+        class="header"
+      >
         <h4>{{ device.devicename }}</h4>
 
         <icon-link
           :icon="['fas', 'angle-right']"
-          :link="{ name: 'device', params: { devicename: device.devicename }}"
-          :position="'right'"/>
+          :link="{ name: 'device', params: { devicename: device.devicename } }"
+          :position="'right'"
+        />
       </router-link>
     </div>
   </div>
 </template>
 
 <script>
 import { mapState } from "vuex";
-import DeviceDetail from "./DeviceDetail.vue";
```

Comment:
Did prettier remove this automatically?

Reply:
No this was me, based on warnings from the new version of eslint.